File: PRJ301/TestAll/Given/Q3_XXX/src/java/model/Question.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.util.ArrayList;

/**
 *
 * @author sonnt
 */
public class Question {
    private String content;
    private ArrayList<Answer> answers = new ArrayList<>();

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public ArrayList<Answer> getAnswers() {
        return answers;
    }

    public void setAnswers(ArrayList<Answer> answers) {
        this.answers = answers;
    }
    
}

File: PRN211/WebApplication1/WebApplication1/Models/Question.cs
namespace WebApplication1.Models
{
    public class Question
    {
        public int Id { get; set; }
        public string Title { get; set; } = null!;
        public string A { get; set; } = null!;
        public string B { get; set; } = null!;
        public string C { get; set; } = null!;
        public string D { get; set; } = null!;
        public string Answer { get; set; } = null!;
    }
}
